docs/adrs/0550-Credscan-for-secret-scan.md

```diff
@@ -0,0 +1,19 @@
+# ADR 0550: Use CredScan for credential scanning
+
+**Date**: 2020-06-22
+
+**Status**: Proposed
+
+## Context
+Currently, there are two scenario's where there are chances that credentials can leak into workflow logs.
+- Action author can unintentionally introduces bug which prints non-github credentials on console.
+- User is using some CLI based action and his/her script can print non-github credentials on console.
```

Comment:
This is also true for `runs` step and you should call that out as it is one of most used steps.